index.js

```diff
@@ -2,11 +2,13 @@
  * @format
  */
 
-import { AppRegistry } from 'react-native';
+import { AppRegistry, LogBox } from 'react-native';
 import App from './App';
 import { name as appName } from './app.json';
 
 if (!__DEV__) {
   console.ignoredYellowBox = ['Warning: Each', 'Warning: Failed'];
 }
+
+LogBox.ignoreLogs(['Persistent storage is not supported on tvOS, your data may be removed at any point.']);
```

Comment:
If we know we're going to find another approach for storage on tvOS, do we need to ignore this warning?

Reply:
I removed it simply because it pops up in the UI and disturbs the experience when testing...

Reply:
OK, understood - can we add a `TODO` to remind us to remove it once we've worked out how to handle persistent storage?

Reply:
That's added now